recipes/libgpiod/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set name = "libgpiod" %}
+{% set version = "2.1.2" %}
+
+package:
+  name: {{ name|lower }}-split
+  version: {{ version }}
+
+source:
+  url: https://git.kernel.org/pub/scm/libs/{{ name }}/{{ name }}.git/snapshot/{{ name }}-{{ version }}.tar.gz
+  sha256: b1bdf1e3f75238695f93e442062bafc069170f2bf4f0cd4b8e049ca67131a1f0
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+outputs:
+  - name: {{ name }}
+    script: build_base.sh
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ stdlib('c') }}
+        - make
+        - autoconf
+        - autoconf-archive
+        - pkg-config
+        - automake
+        - libtool
+    test:
+      commands:
+        - test -f "${PREFIX}/lib/libgpiod.so"
+        - test -f "${PREFIX}/include/gpiod.h"
+    about:
+      license: LGPL-2.1-or-later AND GPL-2.0-or-later
+      license_file:
+        - LICENSES/LGPL-2.1-or-later.txt
+        # used in gpio-tools
```

Comment:
If this also includes tools, can you test that they are correctly installed? Or even better, split them into their own package to make the library purely `LGPL`

Reply:
done in [`cd567c0` (#26443)](https://github.com/conda-forge/staged-recipes/pull/26443/commits/cd567c031e9f105d6ff63fef614c5f483cc63342)